### PR TITLE
Fix Bug 1371448: Remove search navigator

### PR DESCRIPTION
**Pull request by stephaniehobson**
Search navigator show up around the page title after you follow a link from the search results page.

I don't have search configured locally so this is untested :(

Things to check:
- assets build
- nothing visually amiss on an article page
- nothing visually amiss after following a link to an article from the search results
- no javascript errors in console during all this.

### CODE REVIEW

File: kuma/settings/common.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Django settings for kuma project.
+`# Django settings for kuma project.
```
Comment (jwhitlock) [inline kuma/settings/common.py]: Extra quote should be removed in front of this comment.

File: kuma/settings/common.py [hunk 2]
```diff
@@ -1591,3 +1589,4 @@ def get_user_url(user):
 ce_path = path('kuma', 'settings', 'content_experiments.json')
 with open(ce_path, 'r') as ce_file:
     CONTENT_EXPERIMENTS = json.load(ce_file)
+`
```
Comment (jwhitlock) [inline kuma/settings/common.py]: And remove this one as well.

Or, triple quotes if you mean to comment out the whole file 😁

Comment (stephaniehobson) [inline kuma/settings/common.py]: Weird. I wonder how that got in there...

**Comment by stephaniehobson** [comment]
Fixed weird ticks.

**Comment by stephaniehobson** [comment]
Remove waffle switch.

**Comment by jwhitlock** [comment]
Dead code, JS, styles _and_ a waffle flag removed. 🥇

**Pull request merged**